### DISCUSSION TITLE
Security Onion 2.3.40 Cases Install Help

### DISCUSSION THREAD

**Discussion by CollinBeder** [Security-Onion-Solutions/securityonion#13351] [Unsupported Versions]
I installed Security Onion 2.3.40 without cases originally, and now I need to install cases. Is there anyway to install cases and what is the process to do so.

**Comment by TOoSmOotH** [comment]
Security Onion 2.3 is EOL and is no longer supported.  Please see:

https://docs.securityonion.net/en/2.4/eol.html

**Reply by CollinBeder** [reply]
@TOoSmOotH I understand that it is EOL, but I'm developing a custom exam environment and need to use version 2.3.40. Do you know if there is a work around to set Cases up on 2.3.40. I am not able to upgrade to 2.4.

**Reply by InfosecGoon** [reply]
Cases wasn't introduced until 2.3.100 -- in 2.3.40, case management was still being done with TheHive.

**Reply by CollinBeder** [reply]
@InfosecGoon Ahh, are you aware of a way to install TheHive after initial installation on 2.3.40?

**Reply by dougburks** [reply]
@CollinBeder You may be able to manually install TheHive but we can't provide any further assistance for TheHive or Security Onion 2.3.
